### Investigate 15710 null assessment_item_response_id

In [6]:
%run '../lib/libraries.ipynb'

dython              0.6.1
tinydb              4.2.0


In [13]:
dfFlt = load_df('dfFlt')

dfNull = dfFlt.filter( F.col('assessment_item_response_id').isNull() )

dfNull.count()

15710

- 15,710  observations

##### Are observations correlated with final_score_unweighted has 16,582 zero values

In [14]:
dfNull.filter(F.col('final_score_unweighted') == 0).count()

13715

- 13715 observations related
- Ignore those observations


##### Observations with final_score_unweighted > 0

In [15]:
dfNull = dfNull.filter(F.col('final_score_unweighted') > 0)
dfNull.count()

1995

- 1,995 observations

In [5]:
finish_todo('Investigate 15710 null assessment_item_response_id')
